dev/drivers/scripts/stats/nfcens/jevs_nfcens_wave_grid2obs_stats.sh

```diff
@@ -19,7 +19,7 @@ export COMPONENT=nfcens
 export STEP=stats
 export RUN=wave
 export VERIF_CASE=grid2obs
-
+export OMP_NUM_THREADS=1
```

Comment:
@SamiraArdani-NOAA Is this line necessary (i.e., is it already covered by OMP_NUM_THREADS = 1 in `parm/metplus_config/stats/nfcens/wave_grid2obs/PointStat_fcstNFCENS_obsGDAS_climoERA5_Wave_Multifield.conf`)? If not, does `export OMP_NUM_THREADS=1` need to be added to the corresponding .ecf script? Only the .ecf scripts (not the /dev/drivers/scripts) will be run in ops. Please let us know. Thanks!

CC @ShelleyMelchior-NOAA @PerryShafran-NOAA

Reply:
@AliciaBentley-NOAA : ok. Let me test the code without adding this line into /dev/drivers/scripts.

Reply:
@AliciaBentley-NOAA : I just tested the code without the highlighted line (line 22 in /dev/drivers/scripts). It shows the warning message that Wei commented on. By having this line in /dev/drivers/scripts, the warning message is gone as I tested yesterday.

Reply:
@SamiraArdani-NOAA Thanks! If this is the case, then please also add export `OMP_NUM_THREADS=1` to the corresponding .ecf script, since that is the script that will be run in ops to kick off this job. Thanks! 

CC @ShelleyMelchior-NOAA @PerryShafran-NOAA